recipes/elegansbot/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "elegansbot" %}
+{% set version = "1.0.2" %}
+{% set python_min = "3.8" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source: 
+  url: https://files.pythonhosted.org/packages/b1/07/3a1453d6e2df7f59e7a5d8fe3a652afbf7d77f23479bddd6767a18c1b0b1/elegansbot-1.0.2.tar.gz
+  sha256: 16ad4cd055b420ea45f42a9d4c914a58405da5373c6a47bac71b67c3807152ca
+
+build:
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+  number: 0
+
+requirements:
+  host:
+    - python {{ python_min }}
+    - pip
+    - setuptools
+  run:
+    - python >=3.8
+    - numpy >=1.19.0
+    - numba >=0.54.0
+    - scipy >=1.5.0
+    - matplotlib-base >=3.4.2
+
+test:
+  imports:
+    - elegansbot
```

Comment:
Please see the following docs about using `python_min` in the `test` section. It would also be advisable to add a `pip check` as well:
https://github.com/conda-forge/staged-recipes/blob/main/recipes/example/meta.yaml#L64-L67